__tests__/auth.spec.js

```diff
@@ -67,4 +69,107 @@ describe('AUTH API ENDPOINTS', () => {
             expect(body).toMatchSnapshot();
         });
     });
+
+    describe('**INTERNET CONNECTION NEEDED** POST PASSWORD RESET api/v1/auth/reset', () => {
```

Comment:
No internet connection should be needed to test. Mock out any libraries that connect to the internet.

Reply:
i now understand that u made a mock of the friendly mail.... i want to know if the mocked version will replace the real one automatically during the test?

Reply:
fixed